packages/nuekit/test/page-router.test.js

```diff
@@ -0,0 +1,165 @@
+import { beforeAll, afterEach, afterAll, test, expect, spyOn, jest } from 'bun:test'
```

Comment:
At least the other tests I've seen, don't have these imports, so I would just leave it out, at least, if it works without them. WDYT?

Reply:
Done.